components/react/traduction/README.de.md

```diff
@@ -0,0 +1,216 @@
+[![banner](https://cdn.matteobruni.it/images/particles/banner2.png)](https://particles.matteobruni.it)
+
+# react-tsparticles
+
+[![npm](https://img.shields.io/npm/v/react-tsparticles)](https://www.npmjs.com/package/react-tsparticles) [![npm](https://img.shields.io/npm/dm/react-tsparticles)](https://www.npmjs.com/package/react-tsparticles)
+
+Offizieller[tsParticles](https://github.com/matteobruni/tsparticles) ReactJS-Komponent
+
+## Installation
+
+```shell
+
+npm install react-tsparticles
+
+```
+
+oder
+
+```shell
+
+yarn add react-tsparticles
+
+```
+
+#### create-react-app
```

Comment:
Codacy found an issue: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)